.gitmodules

```diff
@@ -1,6 +1,8 @@
 [submodule "qemu"]
 	path = qemu
-	url = https://github.com/pranith/qemu
+	url = https://github.com/B-Bimmermann/qemu-1.git
+	branch = qsim_v25_qemu_26
```

Comment:
You need to create a branch with the changes you want upstream and send a pull request from that branch. You can maintain another branch or your own master branch with the changes which are specific to you, like this change for example.